js/deer-config.js

```diff
@@ -304,7 +304,12 @@ export default {
                         return
                     }
                     queryString = queryString.trim()
-                    const query = decodeContentState(queryString)
+                    const query = decodeContentState(queryString);
+                    for (const key in query) {
```

Comment:
We don't want to be altering the original query data. Something else may need it to be case sensitive. Do not change query.  I am surprised it let you, you will notice it is defined with a `const` and so should not even be allowed to change.

Reply:
remove the `;`

Reply:
This is the extra complexity that should be removed.